<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_EditObjectDatabase</class>
 <widget class="QWidget" name="form_EditObjectDatabase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lw_objects_chooser">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_objectdb_new">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_objectdb_remove">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_objectdb_apply">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_objectdb_path">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_objectdb_txt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_objectdb_icon_path">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_path">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Not set</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_icon_path">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Not set</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_path_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_path_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_path_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Icon path</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_objectdb_path_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pt_objectdb_description">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_test">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>llllabel</string>
   </property>
  </widget>
  <zorder>lbl_test</zorder>
  <zorder>lw_objects_chooser</zorder>
  <zorder>pb_objectdb_new</zorder>
  <zorder>pb_objectdb_remove</zorder>
  <zorder>pb_objectdb_apply</zorder>
  <zorder>pb_objectdb_path</zorder>
  <zorder>le_objectdb_txt</zorder>
  <zorder>pb_objectdb_icon_path</zorder>
  <zorder>lbl_objectdb_path</zorder>
  <zorder>lbl_objectdb_icon_path</zorder>
  <zorder>lbl_objectdb_path_3</zorder>
  <zorder>lbl_objectdb_path_4</zorder>
  <zorder>lbl_objectdb_path_5</zorder>
  <zorder>lbl_objectdb_path_6</zorder>
  <zorder>pt_objectdb_description</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
